site/pages/search-results-contextual-en.hbs

```diff
@@ -0,0 +1,111 @@
+---
+{
+	"title": "Search results",
+	"language": "en",
+	"altLangPrefix": "search",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" }
+	],
+	"dateModified": "2018-12-21",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<!--Note: visual example with styles applied: https://gc-proto.github.io/student-loans/scenarioA/services/search-results-contextual-visual.html-->
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<div class="row profile">
+	<div class="col-md-12">
+		<h1 property="name">Search results</h1>
+		<div class="well">
+			<form name="cse-search-box" method="GET" action="/en/sr/srb.html" id="cse-search-box" role="form">
+				<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
+				<input type="hidden" name="st" value="s" />
+				<input type="hidden" name="s5bm3ts21rch" value="x" />
+				<input type="hidden" name="num" value="10" />
+				<input type="hidden" name="st1rt" value="0" />
+				<input type="hidden" name="langs" value="eng" />
+				<input type="hidden" name="cdn" value="canada" />
+				<input type="hidden" name="hq" value="" />
+				<div class="input-group mrgn-tp-lg">
+					<input value="[value]" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" class="form-control" type="search">
+					<span class="input-group-btn">
+						<button type="submit" id="wb-srch-sub" class="btn btn-primary btn-small" name="wb-srch-sub"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search</span></button>
+					</span>
+				</div>
+			</form>
+			<div class="row mrgn-tp-md">
+				<div class="col-md-4 nowrap">
+					<input type="radio" name="searchscope" id="radio-allresults" value="allresults"  /> <label id="label-alldepartments" for="radio-alldepartments">All Government of Canada websites</label>
```

Comment:
Did you try with an inline unordered list?

Reply:
Not sure if I got the syntax right - commiting it

Reply:
The markup change you did seems ok.